kube-hunter.py

```diff
@@ -20,8 +20,10 @@
 parser.add_argument('--statistics', action="store_true", help="set hunting statistics")
 
 import plugins
+import conf
 
 config = parser.parse_args()
+conf.init(config)
```

Comment:
After renaming `conf` to `config`, you can either use the `args` variable name or directly init with `parse_args` return value